enzyme/benchmarks/ReverseMode/ba/Makefile.make

```diff
@@ -16,8 +19,8 @@ clean:
 	opt $^ -o $@ -S
 	#opt $^ -O2 -o $@ -S
 
-ba.o: ba-opt.ll
-	clang++ -O2 $^ -o $@ $(BENCHLINK)
+ba.o: ba.cpp
+	clang++ $(LOAD) $(BENCH) ba.cpp -I /usr/include/c++/11 -I/usr/include/x86_64-linux-gnu/c++/11 -O2 -o ba.o -lpthread $(BENCHLINK) -lm -L /usr/lib/gcc/x86_64-linux-gnu/11
```

Comment:
So as is this prevents from running the original enzyme ablation hypotheses, can you retain a way to do so?

Reply:
Does the lstm example work for you? Then I can move all over to that.